packages/experimental/Switch/src/Switch.tsx

```diff
@@ -14,9 +41,48 @@ export const Switch = compose<SwitchType>({
     toggleContainer: View,
     onOffText: Text,
   },
-  useRender: (_userProps: SwitchProps) => {
-    return (_final: SwitchProps) => {
-      return null; // Only implemented for Win32
+  useRender: (userProps: SwitchProps, useSlots: UseSlots<SwitchType>) => {
+    const animateSwitchKnob = () => {
+      if (Platform.OS === 'android') {
+        LayoutAnimation.configureNext(LayoutAnimation.Presets.easeInEaseOut);
+      }
+    };
+
+    const switchInfo = useSwitch({
+      ...userProps,
+      onChange: (e: InteractionEvent, checked?: boolean) => {
```

Comment:
This param and function would be regenerated on every render.

This code could go into useSwitch directly, we could add a local onChange handler in there.

Reply:
Moved to useSwitch.

Reply:
Changes done and approved.